packages/salesforcedx-sobjects-faux-generator/src/constants.ts

```diff
@@ -6,13 +6,10 @@
  */
 
 export const CLIENT_ID = 'sfdx-vscode';
-export const SFDX_DIR = '.sfdx';
-export const TOOLS_DIR = 'tools';
 export const SOBJECTS_DIR = 'sobjects';
 export const STANDARDOBJECTS_DIR = 'standardObjects';
 export const CUSTOMOBJECTS_DIR = 'customObjects';
 export const SOQLMETADATA_DIR = 'soqlMetadata';
-export const SFDX_PROJECT_FILE = 'sfdx-project.json';
```

Comment:
I got off track a bit originally and attempted to move all these dir references out of the faux package, but then figured out that was creeping to a huge PR so rolled back to just the dirs specific to the config.